.github/scripts/tests/test_veristat_compare.py

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env python3
+
+import unittest
+from typing import Iterable, List
+
+from ..veristat_compare import parse_table, VeristatFields
+
+
+def gen_csv_table(records: Iterable[str]) -> List[str]:
+    return [
+        ",".join(VeristatFields.headers()),
+        *records,
+    ]
+
+
+class TestMatrix(unittest.TestCase):
```

Comment:
not sure if TestMatrix is the most descriptive name for this test class. I believe this is the remnant of some copy/paste.

Reply:
Renamed into `TestVeristatCompare`